lib/tokens/session_token.js

```diff
@@ -24,11 +24,7 @@ module.exports = (log, inherits, Token, config) => {
     this.locale = details.locale || null
     this.mustVerify = !! details.mustVerify || false
 
-    if (details.createdAt > 0) {
-      this.accountCreatedAt = details.createdAt
-    } else {
-      this.accountCreatedAt = null
-    }
+    this.accountCreatedAt = details.accountCreatedAt || null
```

Comment:
Cool! I think there'll be some `account_created_at` that can be banished too, on the event object itself.